src/uris/deepLinks/deepLinkService.ts

```diff
@@ -1386,6 +1402,25 @@ export class DeepLinkService implements Disposable {
 						repository: repo,
 						source: 'launchpad',
 					} satisfies ShowWipArgs);
+					if (this._context.prId) {
+						action = DeepLinkServiceAction.OpenAllPrChanges;
+					} else {
+						action = DeepLinkServiceAction.DeepLinkResolved;
+					}
+					break;
+				}
+				case DeepLinkServiceState.OpenAllPrChanges: {
+					const pr = await this.getPullRequest(this._context.prId!);
+					const refs = getComparisonRefsForPullRequest(this._context.repoPath!, pr!.refs!);
```

Comment:
We should not be forcing assertions on the existence of these values. Following the pattern we use in other states, we should check the existence of the values and properties we need, and if we cannot find them, kick out into the error state with a message.

Reply:
done